includes/required.php

```diff
@@ -2,9 +2,11 @@
 session_start();
 
 require_once(dirname(__FILE__) . '/constant_values.inc.php');
-require_once(dirname(__FILE__) . '/../config.inc.php');
-if(file_exists(dirname(__FILE__).'/config-overrides.php')){
-    include_once dirname(__FILE__).'/config-overrides.php';
+
+// carica le configurazioni di default e quelle personali se esistono
+require_once(dirname(__FILE__) . '/../configs/config.core.inc.php');
+if(file_exists(dirname(__FILE__).'/../config.inc.php')){
+    include_once dirname(__FILE__).'/../config.inc.php';
```

Comment:
```suggestion
require_once(dirname(__FILE__, 2) . '/configs/config.core.inc.php');
if(file_exists(dirname(__FILE__, 2) . '/config.inc.php')){
    include_once dirname(__FILE__, 2) . '/config.inc.php';
```
Suggerimento personale: lo trovo un approccio più pulito perché genera il path corretto senza bisogno di risalire con i `..`. 

Lascio a tua discrezione se accettare il suggerimento, di base già funziona perfettamente come hai fatto 💪